src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,41 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_AGE = 18;
+
+    public static final int MIN_LOG = 6;
+
+    public static final int MIN_PASS = 6;
+
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws UserRegistrationException {
+        if (user.getAge() == null || user.getPassword() == null || user.getLogin() == null) {
```

Comment:
it's better to use separate methods 
validate login
validatePassword
validateAge